sdk/Trace/SpanProcessor/BatchSpanProcessor.php

```diff
@@ -126,7 +126,14 @@ protected function queueReachedLimit(): bool
 
     protected function enoughTimeHasPassed(): bool
     {
-        $now = $this->clock->timestamp();
+        $now = (int) ($this->clock->timestamp() / 1e6);
```

Comment:
delay is in milliseconds and the current must be converted in milliseconds to.

Reply:
I'm wondering if we should add a helper for this conversion in 
https://github.com/open-telemetry/opentelemetry-php/blob/master/sdk/Trace/Clock.php

Not really anything important.

Reply:
I will think of it, and I will open a different PR for that.

Reply:
"time elapsed" is the kind of thing that should use the monotonic clock. The realtime clock can move backwards; the monotonic clock will not (assuming the OS provides a good one, can still happen on some platforms).

Reply:
Good catch. I will use now() instead of timestamp().